minecraft_servers/gameup/manifest.json

```diff
@@ -0,0 +1,91 @@
+{
+  "server_name": "gameup",
+  "nice_name": "§8§l[§c§lGame§f§lUP§8§l]§8",
```

Comment:
I'm not sure right now if colorcodes are okay in the name...

Reply:
Also we can test that too